src/acl.c

```diff
@@ -2042,7 +2388,7 @@ void aclCommand(client *c) {
             return;
         }
 
-        addReplyMapLen(c,5);
+        addReplyMapLen(c,6);
```

Comment:
it's odd (and dangerous) that the map len is set here, but some of the fields are actually part of aclGetUserAddSelectorReply.
i would suggest to use differed reply and have aclGetUserAddSelectorReply return the count, but i know it comes with a performance overhead (many small packets and `write` system calls).
so maybe a comment can resolve that...

Reply:
I don't think this function needs to be high performance, I'm happing doing the deferring.